We enable audit in sudo, but illumos doesn't support AUE_sudo audit event
https://www.illumos.org/issues/8309

--- sudo-1.8.20p2/plugins/sudoers/bsm_audit.c.1	2017-06-02 12:28:31.893829922 +0300
+++ sudo-1.8.20p2/plugins/sudoers/bsm_audit.c	2017-06-02 12:28:57.159739010 +0300
@@ -84,7 +84,7 @@
 	    if (rc == -1)
 #endif
 
-		sudo_warn("au_preselect");
+		/* sudo_warn("au_preselect");*/
 	}
         debug_return_int(rc);
 }
